### DISCUSSION TITLE
Motivation for RDF at 55+

### DISCUSSION THREAD

**Discussion by Wayran-is-typing** [chromiecraft/chromiecraft#3245] [Suggestions]
RDF is dead at 55+.. a lot of my guildies had to wait between 30mins to 1hour for one pop..

Maybe a little reward like 5 chromie points each rdf for healers and tanks (i'm dps btw) could make a difference

**Comment by expigator** [comment]
Agreed, seems in line with my proposal in this list. Some sort of reason to keep raid logging players in game! Honor, rep, tokens, whatever it may be!

**Reply by Wayran-is-typing** [reply]
Hi, expigator! Just i'm thinking on the new players joining CC as a dps and when they reached lvl 55+ and they start trying to play the endgame dungeons for get some gear stuff and must be frustrating can't find a healer or a tank for do a single dungeon (basically cause all the old players who enjoyed the 50-59 bracket got all the gear from dungeons a lot of time ago and there is nothing to do on rdf for them except to help a guildie **ofc in a premade group**)

**Reply by drislands** [reply]
What about Emblems of Heroism, with the only purchasable options being the Heirloom items? That could be a great boon for folks that have Heal/Tank mains and want to get leveling gear for DPS alts.

Though I suppose that would need to be modified from the original Wrath implementation, since IIRC those scale all the way up to level 80 and might outclass some gear at cap.

**Comment by Nyeriah** [comment]
Hey,

This is a cool idea and we might do something like this in the future, but I don't have any further information to share at this time. 

Thanks for the suggestion, we will consider it and analyze what would be the best course of action to offer incentives to increase the dungeon activity.

Regards.

**Reply by Santaria94** [reply]
It's been three months already. How is the analysis progressing and is it worth waiting for something to solve this problem? RDF looks dead and it's very sad for new players and alt-character owners
